backend/src/main/java/hu/flowacademy/worksheet/service/UserService.java

```diff
@@ -30,6 +30,15 @@ public User saveUser(@NonNull User user) throws ValidationException {
         return userRepository.save(user);
     }
 
+    public User update(Long id, User user) throws ValidationException {
+        validateUpdate(user);
+        User updatedUser = userRepository.findById(id).get();
```

Comment:
Ez így nem túl biztonságos (NullPointerException gyanús). Kezeld azt is, ha nem található id alapján (orElseThrow(...)), ami dobjon validációs hibát, illetve loggod, hogy milyen id-val történt a hibás lekérés, a log level legyen error.